src/main/java/com/qaprosoft/carina/demo/mobile/gui/swaglabapp/common/AllertScreenBase.java

```diff
@@ -0,0 +1,13 @@
+package com.qaprosoft.carina.demo.mobile.gui.swaglabapp.common;
+
+import org.openqa.selenium.WebDriver;
+
+public abstract class AllertScreenBase extends AbstractMobilePageBase {
```

Comment:
Typo `Allert` --> `Alert`

Reply:
Also you can face other alerts on iOS. Specify that this alert is for camera access. For example, `CameraAccessAlertBase`